### PR TITLE
changed docs for generate ssh key location

### PR DESCRIPTION
**Pull request by williexu**
Fixes https://github.com/Azure/azure-cli/issues/2376

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ na] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ na] Each command and parameter has a meaningful description.
- [ na] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):  # pylin
 
 for scope in ['vm', 'vmss']:
     register_cli_argument(scope, 'no_auto_upgrade', action='store_true', help='by doing this, extension system will not pick the highest minor version for the specified version number, and will not auto update to the latest build/revision number on any scale set updates in future.')
-    register_cli_argument('{} create'.format(scope), 'generate_ssh_keys', action='store_true', help='Generate SSH public and private key files if missing', arg_group='Authentication')
+    register_cli_argument('{} create'.format(scope), 'generate_ssh_keys', action='store_true', help='Generate SSH public and private key files if missing. Generally will be stored in the ~/.ssh directory', arg_group='Authentication')
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py]: Generally?

**Comment by tjprescott** [comment]
Also, if this is in response to an issue, put something like "Fixes #<whatever>" in the PR description. That way when the PR is merged the issue will automatically be closed.

**Comment by derekbekoe** [comment]
@williexu The format wasn't recognized by GitHub. See https://help.github.com/articles/closing-issues-using-keywords/

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py (hunk 1 above)
Comment (derekbekoe) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py]: I would agree with Travis.
According to https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py#L685, it's always generated under that directory.
Maybe 'The keys are generated under the ~/.ssh directory.' instead?

Comment (derekbekoe) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py]: Also, consider adding this change to the ACS module as `generate_ssh_keys` exists there also.

**Pull request merged**